plugin.video.retrospect/channels/plugin.video.retrospect.channel.be/een/chn_een.json

```diff
@@ -0,0 +1,15 @@
+{
+  "channels": [
+    {
+      "guid": "16A5C86D-52C5-48C1-B894-1CC23EBF1CD8",
+      "name": "Eén",
+      "description": "Online uitzendingen van www.een.be.",
```

Comment:
The thing is, that this is a Belgium channel that is completely focused on Dutch speaking users/viewers. So that is the reason why it is all in the native language of the channel. The same goes for the others.

Reply:
Keep it in mind for future updates.